### ISSUE TITLE
Error: An internal error has occurred

### ISSUE THREAD

**Issue by bgadhvi1** [Microsoft/AL#4142]
So I'm working with a cloud Business Central sandbox, trying to publish the very simple Hello World extension but receiving the error:

![image](https://user-images.githubusercontent.com/44622284/47750714-e413d200-dc87-11e8-8ab6-3f0b028b34d8.png)

I'm able to authenticate and download the symbols, create the package, but when publishing the extension it gives the error above, I'm not sure what I'm doing wrong, or if the issue is with the build. Yes I'm a newbie to BC so slow learning...

To reproduce try to publish any extension to a Microsoft cloud sandbox.

AL Version is 2.0.48254
BC Version is GB Dynamics NAV 13.0 (25432)

**Comment by JohanStenberg100** [comment]
Hi, can you confirm you have signed up for a business central sandbox?

**Comment by bgadhvi1** [comment]
Hi John, yep, the client has a production licence and we've created a sandbox from production. The sandbox I can access no issues through the Web client, but through AL its proving impossible to publish even the Hello WorId extension.

**Comment by JohanStenberg100** [comment]
@bgadhvi1 thanks for the prompt response, we will investigate.

**Comment by bgadhvi1** [comment]
Thanks Johan, will keep trying out different things in AL to see if it makes any difference. FYI I've reset the sandbox multiple times, cleared credential cache and its made no difference. Permissions on the sandbox look correct as well.

**Comment by bgadhvi1** [comment]
Hi Amol, there doesn't seem to be anything in event viewer and liucences are correct as far as I know. Weird but its actually "half" working now, so when publishing Hello World, it still errors in AL but its actually published to the Sandbox, I can see it in BC and it works, so something has changed since yesterday...

**Comment by TravisChe** [comment]
I'm having the similar issue here. I'm using Business Central Cloud Sandbox. When I try to publish my app facing the same error. Any solution for this?
<img width="642" alt="capture" src="https://user-images.githubusercontent.com/40016152/47839032-26562400-ddec-11e8-9c9f-acd1ad9b7eb7.PNG">

**Comment by bgadhvi1** [comment]
Hi Travis, my issue is still ongoing and have not heard anything around the resolution for this, but the extension is still published to the sandbox environment even though it errors in AL, perhaps check your sandbox and double check it definitely hasn't been published.

**Comment by TravisChe** [comment]
@bgadhvi1 You're right! The package is published successfully and the changes is reflected to the sandbox. At least I can continue my development now. Thanks for the sharing 😄

**Comment by AndreyKorepanov** [comment]
Hi @bgadhvi1, thank you for reporting the issue. It seems to be the VS Code problem, we recently fixed. Could you try it with the latest insider version of VS Code, it should fix the problem.

**Comment by bgadhvi1** [comment]
Hi @AndreyKorepanov 

Awesome, yep, works fine on VS Code Insiders!
